### PR TITLE
dm: remove useless log print

### PR DESCRIPTION
**Pull request by Ehco1996**
do not need this log in test

### CODE REVIEW

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

File: jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy [hunk 1]
```diff
@@ -231,7 +231,7 @@ def run_single_it_test(String case_name) {
                      emptyDirVolume(mountPath: '/home/jenkins', memory: true)]) {
         node(label) {
             println "${NODE_NAME}"
-            println "debug command:\nkubectl -n jenkins-tidb exec -ti ${env.NODE_NAME} bash"
+            println "debug command: \n kubectl -n jenkins-tidb exec -ti ${env.NODE_NAME} -c golang bash"
```
Comment (lance6716) [inline jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:234]:
```suggestion
            println "debug command: \nkubectl -n jenkins-tidb exec -ti ${env.NODE_NAME} -c golang bash"
```

File: jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy [hunk 2]
```diff
@@ -242,6 +242,8 @@ def run_single_it_test(String case_name) {
                                 # use a new version of gh-ost to overwrite the one in container("golang") (1.0.47 --> 1.1.0)
                                 export PATH=bin:$PATH
 
+                                sudo yum update && sudo yum install -y procps python
```
Comment (lance6716) [inline jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:245]: don't just think, always test the changes. I remember the base image is Debian

Comment (Ehco1996) [reply jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:245]:
```
describe pod show that:
  golang:
    Container ID:  docker://1012c589d76ec6a91b393525fafcfdeb2ff8dbf5dff2144a28bfd80503a21a0a
    Image:         hub.pingcap.net/pingcap/centos7_golang-1.16:latest
    Image ID:      docker-pullable://hub.pingcap.net/pingcap/centos7_golang-1.16@sha256:ffa545ced82ae2fb798f7d9b28992f7b84a8289c42a4a546586006892ef5be47

I'm not sure if it's always this centos image,Let me check with the relevant guys
```

Comment (Ehco1996) [reply jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:245]: oh, i am sure we use centos as base image, in L55 `POD_GO_DOCKER_IMAGE = 'hub.pingcap.net/pingcap/centos7_golang-1.16:latest'`

Comment (Ehco1996) [reply jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:245]: BTW, **already** tested this ci change in 
https://ci.pingcap.net/blue/organizations/jenkins/dm_ghpr_new_test/detail/dm_ghpr_new_test/691/pipeline*

Comment (lance6716) [reply jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:245]: I didn't logged into the right pod 😂 not using `-c golang`

And seems the centos image has installed `ps` and `python`

**Review by purelind — APPROVED** [review]
LGTM

**Pull request merged**